README.md

```diff
@@ -74,6 +74,22 @@ See the [oauth2 docs][] for complete instructions on using that library.
 For API methods that require HTTP Basic Authentication, use the
 [`BasicAuthTransport`](https://godoc.org/github.com/google/go-github/github#BasicAuthTransport).
 
+GitHub Apps authentication can be provided by [ghinstallation](https://github.com/bradleyfalzon/ghinstallation).
+
+```go
+import "github.com/bradleyfalzon/ghinstallation"
+
+func main() {
+  // Wrap the shared transport for use with the integration ID 1 authenticating with installation ID 99.
+  itr, err := ghinstallation.NewKeyFromFile(http.DefaultTransport, 1, 99, "2016-10-19.private-key.pem")
+  // Handle error.
```

Comment:
I meant like this:

```Go
itr, err := ghinstallation.NewKeyFromFile(http.DefaultTransport, 1, 99, "2016-10-19.private-key.pem")
if err != nil {
	// Handle error.
}
```

I think it's a lot more readable at a glance this way, compared to just a plain text comment without the `if err != nil {` line.